### DISCUSSION TITLE
is comfyUI server down?

### DISCUSSION THREAD

**Discussion by SariGee** [comfyanonymous/ComfyUI#3562] [General]
anyoneelse having issues with accessing 

 127.0.0.1:8188  ?

**Comment by gtx155** [comment]
You need to be running ComfyUI to access the UI by going to `127.0.0.1:8188`. There isn't a central server for ComfyUI, meaning that if you close the program(CMD is closed) you will not be able to access it.

**Reply by SariGee** [reply]
thank you. i figured it out before seeing this but thanks tons for the reply.new to this. appreciate it.
